### PR TITLE
fix(material-docs-app): remove absolute positioning of all content

### PR DESCRIPTION
**Pull request by Splaktar**
- preserve behavior of not having a scrollbar next to the toolbar
- hide theme picker on small devices (iPhone 5) so that navbar doesn't wrap

Screen Shot 2020-07-21 at 18 51 33

### CODE REVIEW

File: src/app/material-docs-app.scss [hunk 1]
```diff
@@ -20,21 +16,14 @@ material-docs-app > app-component-sidenav {
   flex: 1;
 }
 
-material-docs-app > app-homepage,
-material-docs-app > app-guides,
-material-docs-app > guide-viewer {
+material-docs-app > router-outlet + * {
```
Comment (mmalerba) [inline src/app/material-docs-app.scss:19]: selectors that end with `*` aren't the most efficient, is there any other way to do this?

File: src/app/shared/navbar/navbar.scss [hunk 2]
```diff
@@ -69,3 +69,10 @@
     margin: 0;
   }
 }
+theme-picker {
```
Comment (mmalerba) [inline src/app/shared/navbar/navbar.scss:72]: nit: newline

File: src/styles/_general.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
+html,
+body {
+  height: 100vh;
+}
```
Comment (mmalerba) [inline src/styles/_general.scss:4]: nit: newline

File: src/app/material-docs-app.scss (hunk 1 above)
Comment (Splaktar) [reply src/app/material-docs-app.scss:19]: I don't think so as these are all going to be custom components, some that start with `app-` (like `app-guides`) and some that don't (like `guide-viewer`. Hopefully the `+` limits the scope of the matching for `*` enough that this isn't too inefficient.

**Comment by Splaktar** [comment]
Conflicts resolved.

File: src/app/material-docs-app.scss (hunk 1 above)
Comment (mmalerba) [reply src/app/material-docs-app.scss:19]: Selectors are parsed right-to-left, so the `+` won't help. Is there some way we could apply a common class to all of them instead?

**Comment by Splaktar** [comment]
The missing 2 points from the LH a11y audit is fixed in PR https://github.com/angular/components/pull/21750.

**Comment by Splaktar** [comment]
The other 2 missing points from the LH a11y audit for the Getting Started Guide are fixed in PR https://github.com/angular/components/pull/21751.

File: src/app/material-docs-app.scss (hunk 1 above)
Comment (Splaktar) [reply src/app/material-docs-app.scss]: Done

**Comment by Splaktar** [comment]
OK, this should be ready to go now.

**Review by mmalerba — APPROVED** [review]
LGTM

**Pull request merged**